tests/aws/services/ssm/test_ssm.py

```diff
@@ -159,6 +159,15 @@ def test_get_parameter_by_arn(self, create_parameter, aws_client, snapshot, clea
         snapshot.match("Parameter", parameter_by_arn)
         snapshot.add_transformer(snapshot.transform.key_value("Name"))
 
+    @markers.aws.validated
+    def test_get_parameter_by_name(self, create_parameter, aws_client, snapshot):
```

Comment:
From what I see it's a bit redundant in that `test_get_secret_parameter` already covers this case, but given the explicit name of the `..._by_arn` test, I think there's definitely value in having this more explicit test available next to it 👍